docs/source/Howtos/Beginner-Tutorial/Part1/Beginner-Tutorial-Building-Quickstart.md

```diff
@@ -170,74 +168,77 @@ You know you want to.
+The main command for shaping your game world is `dig`. For example, if you are standing in Limbo, you can dig a route to your new house location like this:
 
     dig house = large red door;door;in,to the outside;out
 
-This will create a new room named 'house'. Spaces at the start/end of names and aliases are ignored so you could put more air if you wanted.  This call will directly create an exit from your current location named 'large red door' and a corresponding exit named 'to the outside' in the house room leading back to Limbo. We also define a few aliases to those exits, so people don't have to write the full thing all the time.
+<!-- This is a confusing paragraph as related to the above command and might be more effective if it were to explicitly identify the use of commas in the arguments. -->
+This will create a new room named "house." Spaces at the start/end of object names and aliases are ignored so you could put more air if you wanted. The above dig command call will directly create an exit from your current location named 'large red door' and a corresponding exit named 'to the outside' in the house room leading back to Limbo. We also define a few aliases to those exits, so people don't have to write the full thing all the time.
 
-If you wanted to use normal compass directions (north, west, southwest etc), you could do that with `dig` too. But Evennia also has a limited version of `dig` that helps for compass directions (and also up/down and in/out). It's called `tunnel`:
+If you wanted to use regular compass directions (north, west, southwest, etc.), you could do that with `dig`, too. However, Evennia also has a specialized version of `dig` that helps with cardinal directions (as well as up/down and in/out). It's called `tunnel`:
 
     tunnel sw = cliff
 
-This will create a new room "cliff" with an exit "southwest" leading there and a path "northeast" leading back from the cliff to your current location.
+This will create a new room "cliff" with a "southwest" exit leading there, and a "northeast" path leading back from the cliff to your current location. <!-- is it always bidirectional be default? if so, we might do well to state so explicitly.>
 
-You can create new exits from where you are, using the `open` command:
+You can create new exits from where you are standing, using the `open` command:
 
     open north;n = house
 
 This opens an exit `north` (with an alias `n`) to the previously created room `house`.
 
-If you have many rooms named `house` you will get a list of matches and have to select which one you want to link to.
+If you have many rooms named `house`, you will get a list of matches and must select to which specifric one you want to link.
 
-Follow the north exit to your 'house' or `teleport` to it:
+Next, follow the northern exit to your "house" by walking north, or you can also `teleport` to it:
```

Comment:
Better to make two sentences here: "... by walking north. You can also `teleport` to it:"

Reply:
Agreed.